### ISSUE TITLE
Not an issue, but a question

### ISSUE THREAD

**Issue by OrionRandD** [zaiste/.doom.d#6]
Hi,
I remember that in the beginning of your doomcasts you used to use light themes, instead of dark themes. I enjoyed that a lot...
This article can be of interest to you... It is a fresh article about the Dark Theme lobby...

https://www.lloydatkinson.net/posts/2024/the-dark-mode-lobby/

Thx for your hardwork on doom-Emacs

**Comment by zaiste** [comment]
@OrionRandD Great article! I totally agree. And this "lobby" is at times too vocal indeed... Thanks for sending the article in!

**Comment by OrionRandD** [comment]
> @OrionRandD Great article! I totally agree. And this "lobby" is at times too vocal indeed... Thanks for sending the article in!

Yeah! The guy is in Mastodon, if you happen to discuss the theme there... I am following him... 
I have tootle here in my debian as a Mastodon client...
